src/features/comentsSlice.ts

```diff
@@ -0,0 +1,72 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+
+type CommentsState = {
+  comments: Comment[];
+  loaded: boolean;
+  hasError: boolean;
+};
+
+const initialState: CommentsState = {
+  comments: [],
+  loaded: false,
+  hasError: false,
+};
+
+export const initComments = createAsyncThunk(
+  'comments/fetch',
+  async (postId: number) => {
+    return getPostComments(postId);
+  },
+);
+
+export const initAddComment = createAsyncThunk(
+  'comments/addComment',
+  async (comment: Omit<Comment, 'id'>) => {
+    return createComment(comment);
+  },
+);
+
+export const initDeleteComment = createAsyncThunk(
+  'comments/removeComment',
+  (commentId: number) => {
+    deleteComment(commentId);
+
+    return commentId;
+  },
+);
```

Comment:
The `initDeleteComment` thunk does not handle errors. Consider adding error handling to manage potential failures when deleting a comment.